common/components/collision-body.hpp

```diff
@@ -22,12 +24,8 @@ namespace tec {
 
 			MotionState& operator=(MotionState&& other) noexcept {
 				transform_updated = other.transform_updated;
-				transform = std::move(transform);
```

Comment:
I removed it because the compiler warned that it was a useless assignment (self-assigning with a move). I now realise, seeing your comment, that I failed to notice that this was because it was meant to be `transform = std::move(other.transform)`! It's not like it was working before anyway but still something that should be fixed.

Reply:
Yeah, we can make an issue to fix this so it doesn't get forgotten if you aren't able to affect this right now. Then we can resolve this.